### PR TITLE
chores + check that postContentApiUrl is defined + remove need for deeplinkFlowId cookie

### PR DESCRIPTION
**Pull request by kulla**

### CODE REVIEW

**Review by LarsTheGlidingSquirrel — CHANGES_REQUESTED** [review]
Added some additional notes for improvements, see below.

Awesome that we now can omit setting another cookie and the code looks more clean now. :+1: 

In my local test environment, I still got an error message: "Die Kommunikation mit dem Remote-System konnte nicht verifiziert werden". Can we be sure that this error does not happen in the production environment? 
image

File: src/backend/server.ts [hunk 1]
```diff
@@ -43,7 +41,7 @@ const nextJsRequestHandler = app.getRequestHandler()
 
 if (isDevEnvironment) loadEnvConfig()
 
-const deeplinkFlowMaxAge = 60
+const deeplinkFlowMaxAge = 20
```
[the remark this answers is not in the document]
Comment (kulla) [reply src/backend/server.ts]: Good catch :smile: -> I changed it to 45min

File: src/backend/server.ts [hunk 2]
```diff
@@ -439,44 +437,60 @@ const server = (async () => {
...
-            'https://purl.imsglobal.org/spec/lti-dl/claim/content_items'
-          ][0].custom
-
-        res
-          .setHeader('Content-type', 'text/html')
-          .send(
-            `<!DOCTYPE html>
+    })
+
+    if (!verifyResult.success) return
```
[the remark this answers is not in the document]
Comment (kulla) [reply src/backend/server.ts:448]: This is done by the `verifyJWT()` method

**Comment by kulla** [comment]
> In my local test environment, I still got an error message: "Die Kommunikation mit dem Remote-System konnte nicht verifiziert werden". Can we be sure that this error does not happen in the production environment?

BTW: We need to ask Daniel how we can fix this. I also have it

File: src/backend/server.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (kulla) [reply src/backend/server.ts:448]: (See https://github.com/serlo/ece-as-a-service/issues/222)

File: src/backend/server.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (kulla) [reply src/backend/server.ts]: See https://github.com/serlo/ece-as-a-service/issues/222

**Review by LarsTheGlidingSquirrel — APPROVED** [review]

**Pull request merged**